src/Language/PureScript/Bundle.hs

```diff
@@ -229,46 +230,20 @@ toModule requirePath mids mid top
     , Just importPath' <- checkImportPath requirePath importPath mid mids
     = pure (Require n importName importPath')
   toModuleElement n
-    | JSVariables var [ varIntro ] _ <- node n
-    , JSLiteral "var" <- node var
-    , JSVarDecl declN (eq : decl) <- node varIntro
-    , JSIdentifier name <- node declN
-    , JSLiteral "=" <- node eq
-    = pure (Member n False name decl [])
-  toModuleElement n
-    | JSExpression (e : op : decl) <- node n
-    , Just name <- accessor (node e)
-    , JSOperator eq <- node op
-    , JSLiteral "=" <- node eq
-    = pure (Member n True name decl [])
-    where
-    accessor :: Node -> Maybe String
-    accessor (JSMemberDot [ exports ] _ nm)
-      | JSIdentifier "exports" <- node exports
-      , JSIdentifier name <- node nm
-      = Just name
-    accessor (JSMemberSquare [ exports ] _ nm _)
-      | JSIdentifier "exports" <- node exports
-      , JSExpression [e] <- node nm
-      , JSStringLiteral _ name <- node e
-      = Just name
-    accessor _ = Nothing
```

Comment:
This is me extracting code out of `toModule` and into `matchMember`, `matchExport`, etc. so that it could be used in both `toModule` and `getExportedIdentifiers`.